WebKitBrowser/WebKitImplementation.cpp

```diff
@@ -260,40 +270,6 @@ namespace Plugin {
         onAutomationSessionRequestNewPage
     };
 
-    static void killHelper(pid_t pid, int sig)
```

Comment:
Please keep killHelper until Thunder supports sending a signal to a particular thread in web process.